index.js

```diff
@@ -1,27 +1,40 @@
 process.env.DISABLE_WEBHOOK_EVENT_CHECK = 'true';
 
 const path = require('path');
-const uuid = require('uuid');
 
 const core = require('@actions/core');
 
-const { Probot } = require('probot');
+const { createProbot: probotCreate } = require('probot');
 
-module.exports = (...handlers) => {
-  // Setup Probot app
-  const githubToken = process.env.GITHUB_TOKEN;
-  const probot = new Probot({ githubToken });
-  probot.setup(handlers);
+function createProbot({
+  overrides = {},
+  defaults = {},
+  env = process.env,
```

Comment:
I think the comment at the end of this line is what the linter is complaining about.